fastserde/avro-fastserde-tests-common/src/test/java/com/linkedin/avro/fastserde/FastGenericSerializerGeneratorTest.java

```diff
@@ -141,7 +141,7 @@ public void shouldWriteFixed() {
   @Test(groups = {"serializationTest"})
```

Comment:
I think that this would be more suitable as a separate PR with corresponding Test Case.

Reply:
Enums-fix will be backed out to https://github.com/RTBHOUSE/avro-util/pull/4

This particular block will be reverted from current PR.